tests/playwright/src/model/pages/main-page.ts

```diff
@@ -94,4 +99,32 @@ export abstract class MainPage extends BasePage {
     }
     return undefined;
   }
+
+  async getRowsFromTableByStatus(status: string): Promise<Locator[]> {
+    await waitUntil(async () => await this.rowsAreVisible(), { sendError: false });
+
+    const table = this.content.getByRole('table');
+    if (await table.isVisible()) {
+      const rows = await table.getByRole('row').all();
+      const filteredRows = [];
+      for (let rowNum = 1; rowNum < rows.length; rowNum++) {
+        //skip header
+        const statusCount = await rows[rowNum].getByRole('cell').nth(2).getByTitle(status, { exact: true }).count();
+        if (statusCount > 0) filteredRows.push(rows[rowNum]);
+      }
+      return filteredRows;
+    }
+    return [];
+  }
+
+  async countRowsFromTable(): Promise<number> {
+    await waitUntil(async () => await this.rowsAreVisible(), { sendError: false });
+
+    const table = this.content.getByRole('table');
+    if (await table.isVisible()) {
```

Comment:
I'm not sure what this if statement's purpose is, if you remove it and the final `return 0` also the code seems to be functionally the same.